sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -956,7 +961,18 @@ object Radar {
                             return
                         }
 
-                        verificationManager.getIntegrityToken(config.googlePlayProjectNumber, config.nonce) { integrityToken, integrityException ->
+                        if (config.nonce == null) {
+                            logger.e("Error missing nonce for trackVerified", RadarLogType.SDK_ERROR)
+                            handler.post {
+                                callback?.onComplete(RadarStatus.ERROR_UNKNOWN)
```

Comment:
@tderouin `ERROR_UNKNOWN` is meant to be a fallback only. In cases where something is missing from the API (e.g., malformed response), we return `ERROR_SERVER` instead